runtime/local/src/chain_extensions.rs

```diff
@@ -18,8 +18,10 @@
 
 //!
 use super::Runtime;
+pub use pallet_chain_extension_assets::weights::WeightInfo;
```

Comment:
Sorry for not commenting this before, but such naming seems to generic for a very specific use case.

Maybe just going without the `use` or using an alias would be better.
Applies to other files too.